### PR TITLE
Fix the logic of getting stderr for readability

### PR DESCRIPTION
**Pull request by ono-max**
Currently, it's challenging to view stderr logs on https://app.launchableinc.com/organizations/ruby/workspaces/debug/data/test-sessions/2611270. This issue arises because the CLI concatenates the `message` and `text` properties. To address this, I've made the following changes:

1. Check if the `text` property exists. If it does, we'll utilize it for stderr since `text` contains more comprehensive information compared to the `message` property.
2. If the `text` property doesn't exist, we'll fallback to using the `message` property for stderr.

### CODE REVIEW

File: tests/data/ant/record_test_result.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
       "duration": 0.006,
       "status": 0,
       "stdout": "",
-      "stderr": "An error message\njunit.framework.AssertionFailedError: An error message\n\tat com.launchable.HelloWorldTest.testWillAlwaysFail(Unknown Source)\n",
```
Comment (ono-max) [inline tests/data/ant/record_test_result.json:18]: `An error message` has been included in the modified text.

File: tests/data/googletest/fail/record_test_result.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
       "duration": 0.001,
       "status": 0,
       "stdout": "",
-      "stderr": "github.com/launchableinc/rocket-car-googletest/test_a.cpp:21\nExpected equality of these values:\n  false\n  true\ngithub.com/launchableinc/rocket-car-googletest/test_a.cpp:21\nExpected equality of these values:\n  false\n  true",
```
Comment (ono-max) [inline tests/data/googletest/fail/record_test_result.json:18]: `github.com/launchableinc/rocket-car-googletest/test_a.cpp:21\nExpected equality of these values:\n  false` has been included in the modified text.

File: tests/data/prove/record_test_result.json [hunk 3]
```diff
@@ -33,7 +33,7 @@
         "duration": 2.0633590221405,
         "status": 0,
         "stdout": "",
-        "stderr": "not ok 1 - add(1, 2) == 5\nnot ok 1 - add(1, 2) == 5\n# Subtest: add\n    not ok 1 - 1 + 2\n    1..1",
```
Comment (ono-max) [inline tests/data/prove/record_test_result.json:36]: `not ok 1 - add(1, 2) == 5` has been included in the modified text.

File: tests/data/prove/record_test_result.json [hunk 4]
```diff
@@ -51,7 +51,7 @@
         "duration": 2.1068799495697,
         "status": 0,
         "stdout": "",
-        "stderr": "not ok 2 - add\nnot ok 2 - add",
```
Comment (ono-max) [inline tests/data/prove/record_test_result.json:54]: `not ok 2 - add` has been included in the modified text.

File: tests/data/pytest/record_test_result.json [hunk 5]
```diff
@@ -42,7 +42,7 @@
        "duration": 0.001,
        "status": 0,
        "stdout": "",
-       "stderr": "assert 1 == False\ndef test_func5():\n  >       assert 1 == False\n  E       assert 1 == False\n\n        tests/funcs3_test.py:6: AssertionError      ",
```
Comment (ono-max) [inline tests/data/pytest/record_test_result.json:45]: `assert 1 == False` has been included in the modified text.

**Review by Konboi — APPROVED** [review]

**Pull request merged**